process_data.py

```diff
@@ -440,6 +449,66 @@ def initialize_result_arrays(lats, lons):
 
     return result_arrays
 
+def flatten_result_arrays(lats, lons, hours, result_arrays):
```

Comment:
why didnt you use the function I provided?
it's way more compact and perfectly suited for the job.

Reply:
I found the recursive approach that is taken in your code very nice and compact as well, but I didn't see how this could be used easily
- defining the dimensions correctly in the "dims" parameter
- (not super important but I did it in my code) keeping the old formatting of variables
I also tried to find a good way now but still hung on that, maybe you see some nice way that I just don't right now. Maybe I'll find a way still...

Reply:
I looked at it quickly last time, based on the name of the function I thought it only did the flattening, but now I see it does more than that.
Please rename the function.

I would first use the flattening function and then do something like this:

result_type = 'ceilings.wind_speed.rank.4'.split('.')
var_name = "_".join([var_name_mapping[at] for at in [result_type[1], result_type[0], result_type[2:]])

Reply:
I tried putting this in the function in Commit b42e19f